src/main/resources/META-INF/microprofile-aliases.properties

```diff
@@ -1,4 +1,6 @@
 # NOTE
 # This file is a placeholder for future aliases of deprecated config settings.
 # Format:
-# dataverse.new.config.option=dataverse.old.deprecated.option
\ No newline at end of file
+# dataverse.new.config.option=dataverse.old.deprecated.option
+dataverse.branding.installation.name=dataverse.settings.fromdb.InstallationName
+dataverse.export.distributor.excludeinstallationifset=dataverse.settings.fromdb.ExportInstallationAsDistributorOnlyWhenNotSet
```

Comment:
It only affects DDI, DDI HTML Codebook, and OpenAire at present but I think it should apply to any export that includes the installation name as a distributor. (The ORE export probably should add it as the intent is for it to be complete  - I followed the json export rather than looking across all the exports when I created the first version.)